### PR TITLE
Fix issue 5466

### PR DESCRIPTION
**Pull request by ChimeraCoder**
@kenpratt and I moved the URL and IFrame fields from Reflow to LayoutTask, as described in #5466.

This is my first attempt at Rust, so let me know how this is!

### CODE REVIEW

File: components/layout/layout_task.rs [hunk 1]
```diff
@@ -135,6 +135,9 @@ pub struct LayoutTask {
     /// The URL of the pipeline that we belong to.
     pub url: Url,
 
+    /// Is the current reflow of an iframe, as opposed to a root window?
+    pub iframe: bool,
```
Comment (Manishearth) [inline components/layout/layout_task.rs]: nit: `is_iframe` might be better (throughout this PR). A parameter of "iframe" sounds like it contains a reference to a frame object

**Comment by Manishearth** [comment]
Aside from a minor nit; this looks good!

I'll wait for @pcwalton or someone to have a quick look since I'm on a phone right now and may have missed something.

Thanks!

**Comment by jdm** [comment]
This is really great! Please make the rename changes that @Manishearth requested, though :)

**Comment by kenpratt** [comment]
@Manishearth @jdm Thanks for the quick reply! I made that change.

**Comment by Manishearth** [comment]
Awesome. Could you squash that commit?

(ignore the warning about history rewrites)

**Comment by ChimeraCoder** [comment]
All three into one commit, or just the name fix (51b257a919817497087d6d9f2c80a21d6fbff795) into the previous (02efc686d55061f98f567d7c5cf2f638e148b826)?

**Comment by Manishearth** [comment]
Whatever you like :smile:

**Pull request merged**